packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -199,7 +213,15 @@ const AccountTransferForm = observer(
             derivez_accounts_to = [];
 
             accounts_list.forEach((account, idx) => {
-                const text = <AccountOption idx={idx} account={account} />;
+                const is_selected_from = account.value === selected_from.value;
```

Comment:
clarification: what if the selected_to account is of pending_verification status? Are we allowing them to deposit?

Reply:
hmmm for this scenario, i dont know specifically, let me ask and clarify

Reply:
so since it is not per account authentication rather it is based on user for eg a user can submit POI and POA using any of the accounts after authentication, both accounts are essentially 'pending' but we only show it for the `selected_from` account and not on `selected_to` 👍